device/common/include/traccc/finding/device/fill_sort_keys.hpp

```diff
@@ -0,0 +1,32 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2024 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "traccc/edm/device/sort_key.hpp"
+#include "traccc/edm/track_candidate.hpp"
+
+namespace traccc::device {
+
+/// Function used for fitting a track for a given track candidates
```

Comment:
The description is a bit misleading unfortunately...

Reply:
Good catch. Fixed